torchtitan/parallelisms/pipeline.py

```diff
@@ -16,7 +16,7 @@
 from torch.distributed.pipelining.stage import PipelineStage
```

Comment:
Now that given the parallelize_llama.py lives under model folder directly, is there a reason to still keep `parallelism` a separate folder instead of merging its module to `components`?